src/opera/test/util/test_metadata_utils.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+#
+# Copyright 2022, by the California Institute of Technology.
```

Comment:
This whole comment block can be removed, as we now maintain a single copyright statement at the top of the repo